client/methods/updateMessage.coffee

```diff
@@ -23,6 +23,7 @@ Meteor.methods
 
 		Tracker.nonreactive ->
 
+			message.editBy = Meteor.userId()
```

Comment:
I think we should consolidate the edit fields.  
message.edit = { userId: Meteor.userId(), ets : new Date...}